src/webgpu/api/operation/compute_pipeline/entry_point_name.spec.ts

```diff
@@ -0,0 +1,10 @@
+export const description = `
+TODO:
+- Test some weird but valid values for entry point name (both module and pipeline creation
```

Comment:
nit: also test using one of many entrypoints in the shadermodule, and creating on compute pipeline from each entrypoint in this large shadermodule (and using them?)